packages/components/src/core/Tag/index.stories.tsx

```diff
@@ -38,13 +39,15 @@ const availableIconOptions = [
 
 const SDS_STYLES: ExtraTagProps["sdsStyle"][] = ["rounded", "square"];
 const COLORS = availableColorOptions.slice(0, 9) as ExtraTagProps["tagColor"][];
+
+// FEEDBACK
+// FEEDBACK
+// FEEDBACK
+// This mutates the constant array, so it's no longer constant
+// Why do we need to dynamically remove the last color?
```

Comment:
PTAL thank you!

Reply:
Well I didn't want to duplicate the color list since it was already declared as a const for use in this file in other tests (e.g. the Default story where user can change the Tag's options from the control panel). This list has two custom color palettes, one with two colors (for if you're not using an icon I guess, based on its name of `labelAndBack`) and one with three colors for it you are using an icon (`labelAndBackAndIcon`). 

However, in the `ScreenshotTest` permutations, each color goes through Tags both with and without icons. the `labelAndBackAndIcon` color palette will just render its first 2 colors if there's no icon, but the `labelAndBack` color palette will leave the icon blank/white since there's not a specific color included for it. Otherwise these two custom color palettes are identical and the permutations they render are redundant (other than the lack of color for icon), so I thought we'd only need to include `labelAndBackAndIcon` palette (since it automatically does the right colors even if there's not an icon). So anyway, this line is excluding the custom color palette that doesn't include a color for icons!

However, I'm not sure if the original color list that `COLORS` copies shoudl instead just be updated to only have the color palette that does include the icon color?